test/test_index.ml

```diff
@@ -44,8 +44,9 @@ let test_active_refs () =
   let hash = "abc" in
   let message = "message" in
   let name = "name" in
-  Index.set_active_refs ~repo
-  @@ Ref_map.singleton "master" { Index.hash; message; name };
+  let ref = { Index.hash; message; name } in
+  let refs = Ref_map.singleton "master" ref in
```

Comment:
I'm not a fan of shadowing `Stdlib.ref`, you could maybe inline the record in the second line?